### PR TITLE
Example functionality

### PR DESCRIPTION
**Pull request by paigeegorry**

### CODE REVIEW

File: src/Sandbox.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      .then(setData)
+      .catch(console.log);
+  }
+
+  return (
+    <section id="endpoint-sandbox">
+      <div>
+        <label htmlFor="endpoint"><h3>Try out an endpoint:</h3></label>
+        <p id="user-input">
+        <span>/api/v1/characters/
```
Comment (Katedam) [inline src/Sandbox.js]:
```suggestion
        <span>/api/v1/characters
```
I think we should remove the slash since some queries don't require it

File: src/App.css [hunk 2]
```diff
@@ -214,9 +213,37 @@ h5 {
...
+  background: black;
+  color: ghostwhite;
+  padding: 10px;
+  width: 200px;
+  border: 2px solid #2f3336
+}
+
+#submit-btn:hover {
+  background: #2f3336;
+  border: 2px solid black;
```
Comment (Katedam) [inline src/App.css]: Can you add `cursor: pointer;` here?

File: src/Sandbox.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    return fetch(`${API_URL}/${userInput}`)
+      .then(res => res.json())
+      .then(setData)
+      .catch(console.log);
+  }
+
+  return (
+    <section id="endpoint-sandbox">
+      <div>
+        <label htmlFor="endpoint"><h3>Try out an endpoint:</h3></label>
```
Comment (Katedam) [inline src/Sandbox.js]:
```suggestion
        <label htmlFor="endpoint-input"><h3>Try out an endpoint:</h3></label>
```

Comment (Katedam) [reply src/Sandbox.js]: For accessibility :-)

File: src/App.css [hunk 4]
```diff
@@ -214,9 +213,37 @@ h5 {
...
   height: 500px;
   overflow: scroll;
-  background-color: ghostwhite;
+  background-color: #2f3336;
+  padding: 5px;
+}
+
+#endpoint-input {
+  background-color: #2f3336;
+  box-shadow: none;
```
Comment (Katedam) [inline src/App.css:229]: The input feels small and hard to see, what if we added some styling like:
```
margin-left: 1em;
width: 300px;
height: 16px;
padding: 4px;
```
The color of the text while typing is very hard to see - I'm wondering about making that lighter too.

Comment (Katedam) [reply src/App.css:229]: Oh also I think lightening the placeholder text would be good!
```
input::placeholder{
  color: #fff;
}
```

File: src/pages/Documentation.js [hunk 5]
```diff
@@ -1,56 +1,135 @@
...
+        classes="endpoints-list"
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+          expandIcon={expandIcon}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+          classes="endpoints-list"
+        >
+          <span className="bold">GET all characters</span> - /api/v1/characters
```
Comment (Katedam) [inline src/pages/Documentation.js]: How do you feel about just having a wide space here instead of the ` - `?
```suggestion
          <span className="bold">GET all characters</span>    /api/v1/characters
```

File: src/pages/Documentation.js [hunk 6]
```diff
@@ -1,56 +1,135 @@
...
-      <p>
-        aliases, otherRelations, affiliation, occupation, residence,
-        appearsInEpisodes, status, gender, eyeColor, born, hairColor,
-        portrayedBy
-      </p>
+      <ExpansionPanel
+        classes="endpoints-list"
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
```
Comment (Katedam) [inline src/pages/Documentation.js]: Would it mess up the Expansion panels to wrap them in li's inside a ul for a more semantic list?

File: src/pages/Documentation.js [hunk 7]
```diff
@@ -1,56 +1,135 @@
...
+              <h3>Available Queries</h3>
+              <p>
+                aliases, otherRelations, affiliation, occupation, residence,
+                appearsInEpisodes, status, gender, eyeColor, born, hairColor,
+                portrayedBy
+              </p>
+            </div>
+            <pre>
+              {JSON.stringify(data.name, null, 1)}
+            </pre>
```
Comment (Katedam) [inline src/pages/Documentation.js]: One thing I noticed on this one is that I thought the scroll wasn't working at first because I had my cursor on top of the queries instead of the code. Could we add some kind of color difference or border or something to show that one section is scrollable and the other isn't?

File: src/tutorial-steps/Six.js [hunk 8]
```diff
@@ -12,7 +12,7 @@ const Six = () => (
       static site like this one to display your documentation.
     </p>
     <p>
-      What should you include? Explain in plain english each route/endpoint, how
+      What should you include? Clearly explain each route/endpoint, how
```
Comment (Katedam) [inline src/tutorial-steps/Six.js:15]: 💅 💯

File: src/App.css [hunk 9]
```diff
@@ -238,25 +265,44 @@ Documentation styles below
...
   display: inline;
   margin-left: 16px;
 }
 
-.endpoints-list > li:hover {
+.endpoints-list > div:hover {
   background-color: #202224;
   border-left: 3px solid #effb31;
   padding-left: 10px;
 }
```
Comment (Katedam) [inline src/App.css]: I'm not seeing this hover state - might need to do some more selective class names. I'm happy to help troubleshoot!

**Comment by Katedam** [comment]
Just a few comments about some style tweaks! Functionality looks great! Would be great to be able to hit enter on the sandbox form instead of clicking the submit button. Doesn't have to be done in this PR tho.

File: src/pages/Documentation.js (hunk 7 above)
Comment (paigeegorry) [reply src/pages/Documentation.js]: Ooh good call. I like the idea of a border. What do you think about pink? And should that exist on the code blocks on the other expansionPanels to be a signal of "scrollable content"?

File: src/App.css (hunk 9 above)
Comment (paigeegorry) [reply src/App.css]: Yeah I'm not seeing what I am missing. It's hard with the material-ui components to capture which component I want to have the hover state. HELP PLZ.

**Comment by paigeegorry** [comment]
Cool, @Katedam ! I pushed up some edits, resolved the ones I fixed and kept open the comments I didn't. Give it another pass when you have time.

File: src/pages/Documentation.js (hunk 7 above)
Comment (Katedam) [reply src/pages/Documentation.js]: Yeah! Sounds good to me! 💅

**Review by paigeegorry — COMMENTED** [review]
Omg bless. 🙏 looks great!
